src/document.rs

```diff
@@ -51,24 +51,22 @@ pub(crate) fn extract_value(any: &PyAny) -> PyResult<Value> {
     if let Ok(b) = any.extract::<Vec<u8>>() {
         return Ok(Value::Bytes(b));
     }
-    if let Ok(dict) = any.downcast::<PyDict>() {
```

Comment:
This looks like an unintentional behaviour change, i.e. before only dictionaries were "depythonized" but everything is.

Reply:
I fixed, I think it's correct?

Reply:
Agreed.